solvebio/resource/apiresource.py

```diff
@@ -326,7 +326,8 @@ def serialize(self, obj):
 
             if isinstance(attr, SolveObject):
                 params[k] = dict(attr)
-            else:
-                params[k] = attr or ""
+                continue
```

Comment:
is this for testing?

Reply:
How do you mean for testing?

Reply:
the latest changes are:
https://github.com/solvebio/solvebio-python/blob/2d71c5a0d91a98ccfa10ebcc556b782d12b303ef/solvebio/resource/apiresource.py#L327

Reply:
Oh I misread the code. Is the continue necessary? Why not an if/else

Reply:
In this case, `continue `does the same as `else`, however, I can change it to be if/else.

Reply:
yes definitely, just feel that if/else is easier to follow